Home.py

```diff
@@ -47,8 +47,8 @@ def home_page():
     ### 📈 Exploratory Data Analysis
     
     - Requires a `csv` file with barcode counts for each sample. This file is generated by `mbarq count` and `mbarq merge` commands.
-    - Also requires a `csv` file describing the experimental design (sample data file)
-    - This page generates an interactive PCA plot and barcode abundance plots for genes of interest
+    - Also requires a `csv` file describing the experimental design. (sample data file)

```

Comment:
Agree on the dot, but should be at the end of the line

Reply:
```suggestion
    - Also requires a `csv` file describing the experimental design (sample data file).

```